src/app/public/modules/autonumeric/autonumeric.directive.ts

```diff
@@ -0,0 +1,46 @@
+import { Directive, ElementRef, Input, OnInit, Optional } from '@angular/core';
+import { AutonumericConfig } from './autonumeric-config';
+let autoNumeric: any = require('autonumeric');
```

Comment:
I'm assuming there aren't `@types` for `autonumeric`?

I know this is inconsequential, but if we could set this variable in PascalCase, it would mimic `autonumeric`'s documentation a little better.

```
const AutoNumeric: any = require('autonumeric');
```

Reply:
Correct, there are no `@types` for `autonumeric`.  I switched this to `const`, but using PascalCase fails a TSLint rule, `variable name must be in lowerCamelCase or UPPER_CASE`.  I can ignore that rule for this line if you think it makes sense.

Reply:
Ah, never mind, then. Thanks!